config/epochConfig.go

```diff
@@ -85,6 +85,7 @@ type EnableEpochs struct {
 	HeartbeatDisableEpoch                             uint32
 	MiniBlockPartialExecutionEnableEpoch              uint32
 	ESDTMetadataContinuousCleanupEnableEpoch          uint32
+	FixAsyncCallBackArgsListEnableEpoch               uint32
```

Comment:
could have extended the test for the `external.toml` parsing

Reply:
done